mapserver/c2cgeoportal.map.mako

```diff
@@ -76,6 +76,7 @@ MAP
     END
 
     LEGEND
+        KEYSIZE 20 20
```

Comment:
@fredj unfortunatly I can't have access to the layers' admin interface (get 404 error, empty table in a random ways).